shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/rulealtered/RuleAlteredJobWorker.java

```diff
@@ -67,8 +64,19 @@
 @Slf4j
 public final class RuleAlteredJobWorker {
```

Comment:
how about this
@NoArgsConstructor(access = AccessLevel.PRIVATE)

Reply:
Thanks, it could be added next time. RuleAlteredJobWorker will be refactored again, there might be no INSTANCE.